xcodeproj/internal/xcodeprojinfo.bzl

```diff
@@ -289,15 +289,19 @@ def _skip_target(
         ),
     )
 
-def _create_test_envs_depset(*, ctx, target, id, automatic_target_info):
+def _create_envs_depset(*, ctx, target, id, automatic_target_info):
     raw_run_env = target[RunEnvironmentInfo].environment if RunEnvironmentInfo in target else {}
     test_env = getattr(ctx.rule.attr, automatic_target_info.env, {})
 
     # Some keys are not applicable in schemes, we will filter them out here
+    _IGNORED_BAZEL_ENV_VARIABLES = [
+        "XCODE_VERSION_OVERRIDE",
+        "XCODE_VERSION"
+    ]
```

Comment:
Let's put this outside of the function, so we don't recreate it each time it's called.

Reply:
Like in a constant in another bzl file?